medusa/server/api/v2/config.py

```diff
@@ -53,13 +53,45 @@ class ConfigHandler(BaseRequestHandler):
     patches = {
         'anonRedirect': StringField(app, 'ANON_REDIRECT'),
         'emby.enabled': BooleanField(app, 'USE_EMBY'),
+        'torrents.authType': StringField(app, 'TORRENT_AUTH_TYPE'),
+        'torrents.dir': StringField(app, 'TORRENT_DIR'),
         'torrents.enabled': BooleanField(app, 'USE_TORRENTS'),
-        'torrents.username': StringField(app, 'TORRENT_USERNAME'),
-        'torrents.password': StringField(app, 'TORRENT_PASSWORD'),
+        'torrents.highBandwidth': StringField(app, 'TORRENT_HIGH_BANDWIDTH'),
+        'torrents.host': StringField(app, 'TORRENT_HOST'),
         'torrents.label': StringField(app, 'TORRENT_LABEL'),
         'torrents.labelAnime': StringField(app, 'TORRENT_LABEL_ANIME'),
-        'torrents.verifySSL': BooleanField(app, 'TORRENT_VERIFY_CERT'),
+        'torrents.method': StringField(app, 'TORRENT_METHOD'),
+        'torrents.password': StringField(app, 'TORRENT_PASSWORD'),
         'torrents.path': BooleanField(app, 'TORRENT_PATH'),
+        'torrents.paused': BooleanField(app, 'TORRENT_PAUSED'),
+        'torrents.rpcurl': StringField(app, 'TORRENT_RPCURL'),
+        'torrents.seedLocation': StringField(app, 'TORRENT_SEED_LOCATION'),
+        'torrents.seedTime': StringField(app, 'TORRENT_SEED_TIME'),
+        'torrents.username': StringField(app, 'TORRENT_USERNAME'),
+        'torrents.verifySSL': BooleanField(app, 'TORRENT_VERIFY_CERT'),
+        'nzb.enabled': BooleanField(app, 'USE_NZBS'),
+        'nzb.username': StringField(app, 'NZBGET_USERNAME'),
+        'nzb.password': StringField(app, 'NZBGET_PASSWORD'),
```

Comment:
These two should be removed (duplicates of `'nzb.nzbget.username'` and `'nzb.nzbget.password'`)